mu-loader.php

```diff
@@ -43,7 +43,7 @@ function mu_loader_plugins_files()
 }
 
 add_action('muplugins_loaded', function(){
-    if (isset($_SERVER['REQUEST_URI']) && strpos($_SERVER['REQUEST_URI'], '/wp-admin/plugins.php') !== false) {
+    if (isset($_SERVER['REQUEST_URI']) && preg_match( '/\/wp-admin(\/network)?\/plugins\.php/i', $_SERVER['REQUEST_URI'])) {
```

Comment:
Since all the file names in wordpress are in lower-case, I think it could be a good idea to remove the case-insensitive flag here, please. Simply for (albeit very minor) performance and security considerations.